netcdfrf.f

```diff
@@ -18,7 +18,13 @@ subroutine netcdfrf(kopt,krf)
       use cqlconf_mod, only : setup0
       use param_mod
       use cqlcomm_mod
-      implicit integer (i-n), real(c_double) (a-h,o-z)
+      implicit none
+      integer kopt,krf ! input
+      integer i,j,ii,is,ll,iray,neltmax,nr, len_ii ! local
+      integer nrays ! nrays will be obtained from reading genray.nc or similar
+      integer length_char, ncvdef2 ! external functions
```

Comment:
I think these should be `external`.  Well, really they should be in a module... but if we're not there yet, we should at least distinguish between numbers and functions that return numbers.